Enterprise-CMCS/macpro-mdct-seds
# Trim packages

### Description
SEDS, and specifically services/ui-src, has accumulated a number of packages and plugins that are no longer in use. This PR removes them, and references to them.

### Related ticket(s)
CMDCT-3540

---
### How to test
Locally, in services/ui-src:
1. `yarn`
2. `yarn test` - all should pass

Additionally, spin up the app with `./run local` and ensure the UI comes up.

If you like, you may also navigate to the user profile page, and confirm that there are no unstyled telephone number inputs (because there are no telephone numbers on the page at all).

Beyond that, the github actions for this PR should succeed.

### Notes
A `yarn` will be prudent (although not immediately necessary) for all developers after pulling in this change.

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._
